ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -226,60 +152,91 @@ The list displays all workflow versions of the selected workflow. Depending on t
-    If you click the start/end place, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+    If you click the start/end place, it becomes highlighted and the corresponding element settings are displayed in the settings sidebar.
 
     ![Start and end place](../../Assets/Screenshots/ActindoWorkFlow/Workflows/StartPlace.png "[Start and end place]")
 
     - *Key*  
-        Place key. This field is read-only. The key is required for API access and must be unique within the workflow version.
+        Place key. This field is read-only. The key is required for API access and must be unique within the workflow version.<!--- Stimmt das, dass er unique sein muss? Beim Import Channels order..ist der Key nicht unique.. p-out-0, p-id-0p-businessDocument-0...-->
 
-    - *Data container*  
+    - *Data container*   
         Use this field to define the data type to be used in the place. If necessary, click the ![Delete](../../Assets/Icons/Cross02.png "[Delete]") (Delete) button to display the whole list of available data types. Enter a search text for the desired data type. The list of transitions is filtered for your search text as you type. Select the radio button of the appropriate option.   
         For detailed information, see [Start and end place](../Overview/04_WorkflowProcessElements.md#start-and-end-place).
 
 - Transition  
     Building blocks that are chained together through places to configure a workflow, represented by a rectangle. All transitions contain a certain, predefined number of input and output ports, represented by a small square to the left and the right of the transition.   
-    When clicking a transition in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+    When clicking a transition in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings sidebar.
 
-    ![Settings side bar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+    ![Settings sidebar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings sidebar]")
 
-    The transition settings side bar is located to the right of the workspace and allows to define the transition settings. The following fields are displayed:
+    The transition settings sidebar is located to the right of the workspace and allows to define the transition settings. The following fields are displayed:
 
     - *Key*  
-        Transition key. This field is read-only. The key is required for API access and must be unique within the workflow version.
+        Transition key. This field is read-only. The key is required for API access and must be unique within the workflow version.<!--- Was ist damit gemeint? und wenn ich eine transition mehrfach einfüge?...-->
```

Comment:
Hier auch, lieber mit Julian/Oli klären, aber ich vermute, es ist gleich wie beim Place. Key muss nur einmal in einer Workflow-Version vorkommen. S. Beispiele unten von Handle delivery note Workflow. Zweimal selbe Transition (Create dispatch note), unteschiedlicher Key:

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/00377abd-b9b0-4801-8bdb-84f88367a125)

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/18f994f2-1628-49f6-9295-73ade11991b9)